src/main/java/uk/gov/register/util/DateFormatChecker.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.register.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+
+import java.text.ParseException;
+
+import static uk.gov.register.util.DateFormat.ISO_8601.*;
+
+public class DateFormatChecker {
```

Comment:
Do we want to test this class also?

Reply:
I think it's repetitive but I've added them